XRTK-Core/Packages/com.xrtk.core/Runtime/Providers/CameraSystem/BaseCameraDataProvider.cs

```diff
@@ -156,18 +156,15 @@ public override void Initialize()
         {
             base.Initialize();
 
-            if (CameraRig == null)
-            {
-                if (CameraCache.Main.transform.parent.IsNull())
-                {
-                    var rigTransform = new GameObject().transform;
-                    CameraCache.Main.transform.SetParent(rigTransform);
-                }
+            cameraSystem.RegisterCameraDataProvider(this);
```

Comment:
Certain this is the line that is initializing

Reply:
Maybe moving this down below that is playing check may help

Reply:
The behaviour for this line has not changed.  It was called in initialise before and is still being called in initialise now.